src/views/QuizView/index.vue

```diff
@@ -0,0 +1,276 @@
+<template>
+  <div
+    v-if="user.isVolunteer && tries < 3"
```

Comment:
Could you add a quick little `<div v-else>` with a message saying something like "no more tries" ?